pkg/devfile/adapters/common/utils_test.go

```diff
@@ -566,14 +566,14 @@ func TestGetCommandsMap(t *testing.T) {
 	for _, tt := range tests {
```

Comment:
Super minor nit: Could we rename the test function from `TestGetCommandsMap` to `TestGetCommands`?

Reply:
Sure.